GitOut/Features/Git/Log/GitTreeEvent.cs

```diff
@@ -158,6 +169,17 @@ private void SetColorAvailable(Color color)
             }
         }
 
+        private bool SetProperty<T>(ref T prop, T value, [CallerMemberName] string? propertyName = null)
```

Comment:
Can remove return value if you want. I'll approve anyway and leave it to you to decice.